msgraphcore/graph_session.py

```diff
@@ -20,7 +20,11 @@ class GraphSession(Session):
     """
     def __init__(self, auth_provider: AuthProviderBase, middleware: list = []):
         super().__init__()
-        self.headers.update({'sdkVersion': 'graph-python-' + SDK_VERSION})
+        if 'sdkVersion' not in self.headers:
+            self.headers.update({'sdkVersion': 'graph-python-' + SDK_VERSION})
+        else:
+            self.headers.update({'sdkVersion': 'graph-python-' + SDK_VERSION + ', '
+                                               + self.headers.get('sdkVersion')})
```

Comment:
Put this in a private method `_append_sdk_version` and call the function in the constructor.

I prefer checking if `sdkVersion` is in `headers` first

```python
if 'sdkVersion' in self.headers:
   ...
else:
  ...
```

Reply:
Sure I'll make the changes!